### PR TITLE
Adding helper functions

### PR DESCRIPTION
**Pull request by IPadawans**
+ Added hydra logs
+ Updated hydra status

### CODE REVIEW

File: scripts/hydra [hunk 1]
```diff
@@ -244,6 +244,21 @@ function update() {
   exit 0
 }
 
+# @cmd Get the logs from containers
+# @arg container_name!        Container name
+# @arg layer!                 Layer name (global-l0, dag-l1, metagraph-l0, currency-l1, data-l1)
+# @option --retroative_rows   Retroative rows from logs file (default 10)
```
Comment (scasplte2) [inline scripts/hydra:250]: minor type (missing `c`) -> `retroactive_rows`

**Review by scasplte2 — APPROVED** [review]
tested via flight-data and works as expected 🔥

File: scripts/hydra (hunk 1 above)
Comment (IPadawans) [reply scripts/hydra:250]: Ooh okay, thanks

File: scripts/hydra [hunk 2]
```diff
@@ -244,6 +244,21 @@ function update() {
   exit 0
 }
 
+# @cmd Get the logs from containers
+# @arg container_name!        Container name
+# @arg layer!                 Layer name (global-l0, dag-l1, metagraph-l0, currency-l1, data-l1)
+# @option --retroactive_rows   Retroactive rows from logs file (default 10)
```
Comment (AlexBrandes) [inline scripts/hydra:250]: I think we should prefer matching the options of tail as closely as possible for this. What about just `-n` here?

Comment (IPadawans) [reply scripts/hydra:250]: Hmm, okay I can do this

Comment (IPadawans) [reply scripts/hydra:250]: @AlexBrandes 
https://github.com/Constellation-Labs/euclid-development-environment/blob/78a418c99a111c845e564455aff5be300550eb1f/scripts/hydra#L247C2-L261C1

**Pull request merged**